airbyte-integrations/connectors/source-chargify/source_chargify/spec.json

```diff
@@ -0,0 +1,21 @@
+{
+  "documentationUrl": "https://docsurl.com",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Chargify Spec",
+    "type": "object",
+    "required": ["api_key", "subdomain"],
+    "additionalProperties": false,
+    "properties": {
+      "api_key": {
+        "type": "string",
+        "description": "Chargify API Key.",
+        "airbyte_secret": true
+      },
+      "subdomain": {
```

Comment:
Do you think you could change this to `domain`? It will allow us to use Chargify's mock server to run our acceptance tests: 
```bash
curl --request GET \
  --url https://stoplight.io/mocks/chargify/api-docs/14108261/customers.json \
  --header 'Authorization: Basic undefined' \
  --header 'Content-Type: application/json'
  ```

Reply:
I change the attribute to use the domain